Frameworks/AEPSwiftUI/Sources/ContentCards/UIElement/AEPImage/AEPImage.swift

```diff
@@ -0,0 +1,60 @@
+/*
+ Copyright 2024 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import SwiftUI
+
+/// The model class representing the image UI element of the ContentCard.
+/// This class handles the initialization of the image from different sources such as URL or bundle.
+public class AEPImage: ObservableObject {
```

Comment:
wondering if we should consider making this `codable`, since it comes from the konductor response in json form

Reply:
I am on the wall about this..
Codable makes things very easy and Infact I think we can initiate the whole Small Image template directly  without writing initializers 

But parsing a dictionary helps me to do some extra validations before I initialize an object..
eg:
 Validating if the string is a URL before initializing an AEPImage
 Secondly in this case, I need to fast fail if the AEPImage does neither URL nor bundled Image

I am losing control on init while using codable.

I will still give this a thought and make a final change later if using codable is more beneficial